motr/client.h

```diff
@@ -1016,6 +1016,8 @@ struct m0_config {
  	 * ADDB size
  	 */
 	m0_bcount_t mc_addb_size;
+
```

Comment:
This variable is not required because we are using environment variable.
local variable can be used instead of configuration structure